test/test_um2netcdf.py

```diff
@@ -70,46 +70,28 @@ def mule_vars(z_sea_rho_data, z_sea_theta_data):
     return um2nc.MuleVars(um2nc.GRID_NEW_DYNAMICS, d_lat, d_lon, z_sea_rho_data, z_sea_theta_data)
 
 
-def set_default_attrs(cube, item_code: int, var_name: str):
-    """Add subset of default attributes to flesh out cube like objects."""
-    cube.__dict__.update({"item_code": item_code,
-                          "var_name": var_name,
-                          "long_name": "",
-                          "coord": {"latitude": 0.0,  # TODO: real val = ?
-                                    "longitude": 0.0},  # TODO: real val
-                          "cell_methods": [],
-                          "data": None,
-                          })
-
-    section, item = um2nc.to_stash_code(item_code)
-    cube.attributes = {um2nc.STASH: DummyStash(section, item)}
-
-
 @pytest.fixture
-def air_temp_cube():
-    # data copied from aiihca.paa1jan.subset file
-    name = "air_temperature"
-    m_air_temp = mock.NonCallableMagicMock(spec=iris.cube.Cube, name=name)
-    set_default_attrs(m_air_temp, 30204, name)
-    return m_air_temp
+def air_temp_cube(lat_river_coord, lon_river_coord):
+    # details copied from aiihca.paa1jan.subset file
+    air_temp = DummyCube(30204, "air_temperature",
```

Comment:
Do these codes have a specific meaning, or do you just need a distinct code for each cube type? If the former, it might benefit from using a reference `dict` or similar rather than having magic numbers.

Reply:
The codes are in the `umpost/stashvar_cmip6.py` module & I've copied them across.

A reference/index dict or constants for common stash codes could certainly help...

Reply:
I've added this as an idea to #27.